csharp/src/main/java/org/jabylon/csharp/CSharpConverter.java

```diff
@@ -69,10 +186,11 @@ public CSharpConverter(URI resourceLocation, boolean prettyPrint) {
 
 	@Override
 	public PropertyFile load(InputStream in, String encoding) throws IOException {
+		LOG.debug("C#:load0");
```

Comment:
these debug statements are pointless.
Also, log messages do not - and should not - contain location information. When the log is written, it contains class, method, and - if wanted - line numbers